ngi_reports/reports/__init__.py

```diff
@@ -106,16 +106,16 @@ def _getfcid(str):
 
     def flatten_dict(self, d, parent_key="", sep="."):
         """
-        Flattens a dict by concatinating the keys
-        seperated by the specified separator (sep) until it finds a value.
+        Flattens a dict by concatenating the keys
+        separated by the specified separator (sep) until it finds a value.
         Optionally it adds parent_key as a prefix.
         E.g. {my : {nested: {key: value}}} -> "my.nested.key: value"
         """
         items = []
-        for k, v in d.iteritems():
+        for k, v in list(d.items()):
```

Comment:
Strictly speaking, you don't need list() when you're just iterating over things, but it's fine to keep it if you want 🙂

Reply:
Done!